otherfiles/RELEASE.md

```diff
@@ -12,7 +12,9 @@
 	1. Add a new header under `Unreleased` section (Example: `## [0.1] - 2022-08-17`)
 	2. Add a new compare link to the end of the file (Example: `[0.2]: https://github.com/sepandhaghighi/mytimer/compare/v0.1...v0.2`)
 	3. Update `dev` compare link (Example: `[Unreleased]: https://github.com/sepandhaghighi/mytimer/compare/v0.2...dev`)
-4. Create a PR from `release` to `dev`
+4. Update `./github/ISSUE_TEMPLATE/bug_report.yml`
```

Comment:
`/.github/ISSUE_TEMPLATE/bug_report.yml`

Reply:
fixed in [6917646](https://github.com/sepandhaghighi/mytimer/pull/80/commits/6917646dcf94b3d4d9cc059949bc17adf3ab95ef).